google-cloud-examples/src/main/java/com/google/cloud/examples/datastore/snippets/CreateEntity.java

```diff
@@ -22,27 +22,38 @@
 
 package com.google.cloud.examples.datastore.snippets;
 
+import com.google.api.gax.rpc.FixedHeaderProvider;
+import com.google.api.gax.rpc.HeaderProvider;
 import com.google.cloud.Timestamp;
 import com.google.cloud.datastore.Datastore;
 import com.google.cloud.datastore.DatastoreOptions;
 import com.google.cloud.datastore.Entity;
 import com.google.cloud.datastore.Key;
 import com.google.cloud.datastore.KeyFactory;
 
-/**
- * A snippet for Google Cloud Datastore showing how to create an entity.
- */
+/** A snippet for Google Cloud Datastore showing how to create an entity. */
 public class CreateEntity {
 
   public static void main(String... args) {
-    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    HeaderProvider headerProvider =
+        FixedHeaderProvider.create("user-agent", "my-client-name", "custom-header", "custom stuff");
```

Comment:
I don't think it makes sense to include a custom header example in all of these samples...

Reply:
Oh, that was accidental. This was just me testing this new functionality....